pkg/cloud/azure/azure.go

```diff
@@ -0,0 +1,211 @@
+package azure
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	submreporter "github.com/submariner-io/admiral/pkg/reporter"
+	"os"
+	"strconv"
+
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/pkg/errors"
+	"github.com/stolostron/submariner-addon/pkg/cloud/reporter"
+	"github.com/stolostron/submariner-addon/pkg/constants"
+	"github.com/submariner-io/cloud-prepare/pkg/api"
+	"github.com/submariner-io/cloud-prepare/pkg/azure"
+	"github.com/submariner-io/cloud-prepare/pkg/k8s"
+	"github.com/submariner-io/cloud-prepare/pkg/ocp"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+const (
+	servicePrincipalJSON = "osServicePrincipal.json"
+	gwInstanceType       = "Standard_D4s_v3"
```

Comment:
We are using a different instance type here compared to the instance type of `Standard_F4s_v2` used as part of  `subctl cloud prepare azure`. Is this a typo or intentional?

Reply:
was a an error when took from a different deployment will make it same as subctl